stories/assets/scss/_typography.scss

```diff
@@ -172,6 +172,29 @@ hr {
   border: 1px solid $color-black;
 }
 
+// link animation
+@keyframes lineLoop-animation {
+  0% {
+    background-position: 100% 100%, -30px 100%;
+    background-size: 100% 2px, 0 2px;
+  }
+
+  to {
+    background-position: calc(100% + 30px) 100%, 0 100%;
+    background-size: 0 2px, 100% 2px;
+  }
+}
+
+a {
+  @extend %paragragh-font-20 !optional;
+
+  color: $color-black;
+  text-decoration: underline;
+  text-decoration-color: red;
+
```

Comment:
@devjgigas , you might want to add here `@extend %animate-link;` placeholder so every link will have animated treatment outside of the box.